variables.tf

```diff
@@ -101,6 +101,12 @@ variable "cluster_tags" {
   default     = {}
 }
 
+variable "tag_cluster_primary_security_group" {
```

Comment:
I propose that instead of using `tag_...`, it is better to prefix with the verb we normally use. For example, `create_tag_cluster_primary_security_group` or `enable_tag_cluster_primary_security_group` or `manage_tag_cluster_primary_security_group`?

Reply:
good point, let me update

Reply:
updated in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/2034/commits/f2dda4002bab0bb0e852bf567a76673ee2d08e73